django_freeradius/api/views.py

```diff
@@ -63,6 +63,12 @@ def post(self, request, *args, **kwargs):
         return getattr(self, method)(request, *args, **kwargs)
 
     def create(self, request, *args, **kwargs):
+        if request.data['session_time'] == '' and request.data['status_type'] == 'Start':
```

Comment:
A more elegant implementation which avoids repetition:

```python
is_start = request.data['status_type'] == 'Start'
for field in ['session_time', 'input_octets', 'output_octets']:
    if is_start and request.data[field] == '':
        request.data[field] = 0
```

Reply:
that's really helpful, thanks!

sometimes I notice repetition, but can't really find a way to solve it. :/